data/binding/gen.go

```diff
@@ -144,9 +145,15 @@ type stringFrom{{ .Name }} struct {
 //
 // Since: 2.0
 func {{ .Name }}ToString(v {{ .Name }}) String {
+{{ if .Format }}
```

Comment:
These added `if`s generate unnecessary newlines in the output, which can be avoided by adding a '-': https://stackoverflow.com/questions/35565915/golang-template-range-newline-removal

Reply:
Nice hint